pallets/subtensor/tests/mock.rs

```diff
@@ -114,6 +114,7 @@ parameter_types! {
 	pub const InitialDefaultTake: u16 = 11_796; // 18% honest number.
 	pub const InitialWeightsVersionKey: u16 = 0; 
 	pub const InitialServingRateLimit: u64 = 0; // No limit.
+	pub const InitialTxRateLimit: u64 = 0; // No limit.
```

Comment:
should we not set a limit right off the bat?

Reply:
What limit should be set? 2 blocks per stake/delegate action?